src/components/skills-quiz/JobCardComponent.jsx

```diff
@@ -29,12 +30,14 @@ const JobCardComponent = ({ jobs, isLoading }) => (
             ) : (
               <>
                 {job.job_postings && job.job_postings.length > 0 && (
```

Comment:
It's not relevant to your work but can we change this condition to
`job.job_postings?.length > 0`

Reply:
Sure, will do.

Reply:
Actually we are already handling this here in your branch https://github.com/edx/frontend-app-learner-portal-enterprise/blob/zafzal/ENT4932/src/components/skills-quiz/JobCardComponent.jsx#L31